packages/cli/src/cli.ts

```diff
@@ -11,11 +12,19 @@ import { setupCliContainer } from './setup-cli-container';
 
 (async () => {
     const scanArguments = (yargs
-        .usage('Usage: $0 --url <url> --output <directoryPath>')
+        .usage('Usage: $0 --url <url> --inputFile <inputFile> --output <directoryPath>')
         .options({
-            url: { type: 'string', describe: 'url to scan for accessibility', demandOption: true },
+            url: { type: 'string', describe: 'url to scan for accessibility' },
+            inputFile: { type: 'string', describe: 'ile path that contains multiple urls separated by newline to scan' },
```

Comment:
fyi: when parsing the urls from the file, you might need to handle \n, \r, \r\n as new line character

Reply:
cool, good to know.